added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/follow/FetchFollow.md

title: Fetch Follows
displayed_sidebar: developSidebar
---

<head>

>   <title>Fetch Followers on Subsocial | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>How To Get Followers With The Subsocial JS SDK</title>
```
